client/src/components/Welcome.js

```diff
@@ -1,10 +1,29 @@
 /* eslint-disable semi */
 import React from 'react';
+import { Link } from 'react-router-dom';
+import Logo from './Logo';
 
-const Welcome = () =>
-    <div className="welcome--container">
-        <h3 className="welcome--message-text">Welcome to TBD :)</h3>
-        <p className="welcome--description-text">Also TBD!</p>
+const Welcome = () => (
+  <div className="welcome--container">
+    <h3 className="welcome--message-text">Welcome to the Template</h3>
+    <div className="page-layout--header">
+      <div className="page-layout--details">
+        <h1 className="page-layout--name">Hack.Diversity MERN Stack Template</h1>
+        <p className="page-layout--description">
+          A simple CRUD app built using the&nbsp;
+          <a
+            href="https://www.educative.io/edpresso/what-is-mern-stack"
+            target="_blank"
+            rel="noopener noreferrer">
+            MERN&nbsp;tech-stack
+          </a>
+          also
+          <Link to="/item/61ddf4aebb70255bc9bd096c/update">Here</Link>.
+        </p>
+      </div>
```

Comment:
Admittedly, I left the "determined" in TBD as "undetermined" 😂 

But this feels too redundant with the `PageLayout` component; if I was new to React, I would be confused as to what purpose each of those components really serves if they're practically identical. I think my original intention was for `PageLayout` to be a kind of container that could take props from a "page" and delegate which prop should render in which part of the overall layout. Since I never got that far and since it feels a little over-engineered for this little app, I would just remove the `PageLayout` component altogether 🤷‍♂️

Reply:
Yeah, I agree